client/src/main/java/io/pravega/client/control/impl/ControllerImpl.java

```diff
@@ -629,6 +629,11 @@ private RetryAndThrowConditionally createRetryConfig(final ControllerImplConfig
         });
     }
 
+    @Override
+    public CompletableFuture<Boolean> createInternalStream(String scope, String streamName, final StreamConfiguration streamConfig) {
+        return this.createStream(scope, streamName, streamConfig);
```

Comment:
This method should not be present in ControllerServiceImpl.... since we do not want users to be able to create internal streams.

Reply:
This was required to be added as in the initial approach we were adding a new method to Controller interface. But now it has been removed